package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "dash-table",
-  "version": "3.1.3",
+  "version": "3.1.4-rc6",
   "description": "Dash table",
-  "main": "build/index.js",
```

Comment:
I'm sure you've explained this before - but what was wrong with using `build/` again?